CustomerApplication/Controllers/CustomerController.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Data.Entities;
+using Data.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace CustomerAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CustomerController : ControllerBase
+    {
```

Comment:
Não vejo nenhuma classe "ControllerBase" no projeto, essa dependência esta correta?
```suggestion
    public class CustomerController
    {
```